tests/storage/blockvolume_test.py

```diff
@@ -112,17 +112,20 @@ def test_fail_invalid_block_volume_size(self, preallocate, vol_format):
 class TestBlockVolumeManifest(VdsmTestCase):
 
     @contextmanager
-    def make_volume(self, size, storage_type='block', format=sc.RAW_FORMAT):
+    def make_volume(self, size, format=sc.RAW_FORMAT, prealloc=sc.SPARSE_VOL):
+        storage_type = 'block'
         img_id = make_uuid()
         vol_id = make_uuid()
         # TODO fix make_volume helper to create the qcow image when needed
         with fake_env(storage_type) as env:
             if format == sc.RAW_FORMAT:
-                env.make_volume(size, img_id, vol_id, vol_format=format)
+                env.make_volume(size, img_id, vol_id,
+                                vol_format=format, prealloc=prealloc)
```

Comment:
We prefer this style when arguments do not fit in one line:

    env.make_volume(
        size, img_id, vol_id, vol_format=format, prealloc=prealloc)

If argument do not fit this way, use one line per argument:

    env.make_volume(
        size,
        img_id,
        vol_id,
        vol_format=format,
        prealloc=prealloc)